modules/runtime/nuxeo-common/src/main/java/org/nuxeo/common/xmap/annotation/XNode.java

```diff
@@ -46,7 +46,7 @@
      *
      * @since 11.5
      */
-    public static final String NO_FALLBACK_MARKER = "__NO_FALLBACK_MARKER__";
+    final String NO_FALLBACK_MARKER = "__NO_FALLBACK_MARKER__";
```

Comment:
You can remove `final` too, it's also implicit.

Reply:
... thanks!